HabboHotel/Users/Ignores/IIgnoresManager.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Plus.HabboHotel.Users.Ignores
+{
+    public interface IIgnoresManager
+    {
+       bool Init(Habbo player);
+       bool TryGet(Habbo userId);
+       bool TryAdd(Habbo userId);
+       bool TryRemove(Habbo userId);
+       ICollection<int> IgnoredUserIds();
+       void Dispose();
+       IReadOnlyCollection<string> GetIgnoredUsers(Habbo uid);
+       Task<Habbo?> IgnoreUser(Habbo uid, Habbo? ignoredid);
```

Comment:
What is the reason for returning `Habbo?` Does seem to make sense to me.
I would only pass the two userIds; `Task IgnoreUser(uint userId, uint targetUserId)`